modules/seqexec/model/src/main/scala/seqexec/model/ModelLenses.scala

```diff
@@ -173,6 +173,9 @@ trait ModelLenses {
   val stringToStepTypeP: Prism[String, StepType] =
     Prism(StepType.fromString)(_.label)
 
+  val stringToBooleanP: Prism[String, Boolean] =
+    Prism[String, Boolean](_.equalsIgnoreCase("true").some)(_.toString)
```

Comment:
I think this should be a `Format` because "TRUE" => `true` => "true" => `true` ...  That is distinct inputs yield the same `getOption` result.  Does it pass the `Prism` tests?

Reply:
Thinking about it a bit more, perhaps it should be a `SplitEpi` because all `String` values map to some `Boolean` value.

```scala
val stringToBoolean: SplitEpi[String, Boolean] =
  SplitEpi(_.equalsIgnoreCase("true"), _.toString)
```

Reply:
Monocle provides a std `stringToBoolean` prism. I'll use that